ui/src/main/webapp/src/app/project/migration-project-form.component.ts

```diff
@@ -49,6 +47,9 @@ export class MigrationProjectFormComponent extends FormComponent implements OnIn
             }
 
             this.isInWizard = flatRouteData.data.hasOwnProperty('wizard') && flatRouteData.data['wizard'];
+
+            if(!this.editMode) // Creating a new project.
+                this._migrationProjectService.deleteProvisionalProjects().subscribe(res => {});
```

Comment:
Yeah, I think we should think about this more after Summit. There are some complications to just keeping it on the server.